Stripe/STPSectionHeaderView.m

```diff
@@ -8,13 +8,46 @@
 
 #import "STPSectionHeaderView.h"
 
+#import "STPAddressViewModel.h"
+#import "STPLocalizationUtils.h"
+#import "STPPaymentConfiguration.h"
+
 @interface STPSectionHeaderView()
 @property(nonatomic, weak)UILabel *label;
 @property(nonatomic)UIEdgeInsets buttonInsets;
 @end
 
 @implementation STPSectionHeaderView
 
++ (instancetype)addressHeaderWithConfiguration:(STPPaymentConfiguration *)config
+                                    sourceType:(STPSourceType)sourceType
+                              addressViewModel:(STPAddressViewModel *)addressViewModel
+                               shippingAddress:(STPAddress *)shippingAddress {
+    STPSectionHeaderView *headerView = [STPSectionHeaderView new];
+    headerView.title = STPLocalizedString(@"Billing Address", @"Title for billing address entry section");
```

Comment:
Billing address is not necessarily the correct text to use on shipping screen if you're filling from SEPA.